qa/tasks/cephfs/test_exports.py

```diff
@@ -150,7 +150,7 @@ def test_session_race(self):
         status = self.fs.wait_for_daemons()
 
         rank1 = self.fs.get_rank(rank=1, status=status)
-        name1 = 'mds.'+rank1['name']
+        _ = 'mds.'+rank1['name']
```

Comment:
if it is not needed maybe delete? or comment out?

Reply:
Done